jupyterhub/tests/selenium/test_browser.py

```diff
@@ -854,3 +857,148 @@ async def test_user_logout(app, browser, url, user):
     while f"/user/{user.name}/" not in browser.current_url:
         await webdriver_wait(browser, EC.url_matches(f"/user/{user.name}/"))
     assert f"/user/{user.name}" in browser.current_url
+
+
+# OAUTH confirmation page
+
+
+@pytest.mark.parametrize(
+    "user_scopes",
+    [
+        (['(no_scope)']),  # no scopes
```

Comment:
Let's keep the user scopes as '[]' here, and handle the missing description from no_scope elsewhere.

```suggestion
        ([]),  # no scopes
```